router/worker.go

```diff
@@ -820,6 +820,7 @@ func (w *worker) sendRouterResponseCountStat(status *jobsdb.JobStatusT, destinat
 		"alert": strconv.FormatBool(alert),
 		// To specify at which point failure happened
 		"errorAt": errorAt,
+		"retry":   fmt.Sprint(status.AttemptNum > 1),
```

Comment:
For consistency with the line above shall we do:
```suggestion
		"retry":   strconv.FormatBool(status.AttemptNum > 1),
```

I am not sure which is more performant, I bet `strconv.FormatBool` would be slightly.
Readability wise `FormatBool` is more explicit, and thus better in my eyes